.github/workflows/validation-comment.yaml

```diff
@@ -1,7 +1,7 @@
 name: Validation Comment
 
 on:
-  pull_request_target:
+  pull_request:
```

Comment:
I don't know why git did this change here on this commit. 
But this re-written to the right value: `pull_request_target` in the next commits.